graphics/gimp3-devel/Portfile

```diff
@@ -150,7 +132,6 @@ destroot.keepdirs   ${destroot}${prefix}/share/gimp/2.99/fonts
 # requires python >= 3.6.0
 
 variant python38 description {Build with python plugin support using python 3.8} {
```

Comment:
please remove variant for EOL Python version

Reply:
Done